lightify/__init__.py

```diff
@@ -962,6 +968,14 @@ def _next_seq(self):
             self.__seq = (self.__seq + 1) % 256
             return self.__seq
 
+    def set_loglevel(self, level):
+        """ set the log level
+
+        :param level: log level (from logging module)
```

Comment:
For consistency you could change description to "logging.loglevel Enum"
```suggestion
        :param level: logging.loglevel Enum
```

Reply:
Otherwise looks good to me.  I will let you know when I test it with Home Assistant.

Thanks!